packages/eas-cli/src/commands/update/index.ts

```diff
@@ -456,6 +461,25 @@ export default class UpdatePublish extends EasCommand {
   }
 }
 
+function assertExpoUpdatesInstalled(projectDir: string, sdkVersion?: string): void {
+  // before sdk 44, expo-update was included in with the expo module
+  if (sdkVersion && semver.lt(sdkVersion, '44.0.0')) {
+    return;
+  }
+
+  const packageJsonPath = resolveFrom.silent(projectDir, './package.json');
```

Comment:
There is util function fo that already https://github.com/expo/eas-cli/blob/main/packages/eas-cli/src/build/utils/updates.ts

If you don't want to import from build consider extracting common util to some common place e.g. https://github.com/expo/eas-cli/blob/main/packages/eas-cli/src/project

Reply:
Agreed with Wojtek, I'd prefer moving the already existing util to top-level utils and use it here instead of duplicating logic.